### ISSUE TITLE
Provide a change tracking support for task with data entries on changed data entries 

### ISSUE THREAD

**Issue by zambrovski** [holunda-io/camunda-bpm-taskpool#354]
If the `DataEntry` is changed via ChangeStream, the update on TaskWithDataEntry should be emited via query emitter.

For data entry change, select all tasks, which have the changed data entry as reference, construct new task with data entry and emit the change

**Comment by zambrovski** [comment]
@lbilger haven't you implemented this during #671 ?

**Comment by lbilger** [comment]
No, it still seems we do not send TaskWithDataEntry updates when a DataEntry changes.
